common/core/desktop/include/keyman/keyboardprocessor.h.in

```diff
@@ -940,7 +941,8 @@ KMN_API
 km_kbp_status
 km_kbp_process_event(km_kbp_state *state,
                      km_kbp_virtual_key vk,
-                     uint16_t modifier_state);
+                     uint16_t modifier_state,
+                     uint8_t is_key_down);
```

Comment:
👏

Reply:
This new signature needs to go into the [15.0 wiki](https://github.com/keymanapp/keyman/wiki/Keyman-15.0-New-Functionality)